samples/quickstart/noxfile.py

```diff
@@ -72,7 +72,7 @@ def get_pytest_env_vars():
     env_key = TEST_CONFIG['gcloud_project_env']
     # This should error out if not set.
     ret['GOOGLE_CLOUD_PROJECT'] = os.environ[env_key]
-    ret['GCLOUD_PROJECT'] = os.environ[env_key]
+    ret['GOOGLE_CLOUD_PROJECT'] = os.environ[env_key]
```

Comment:
can we delete one of these lines?

Reply:
(there's a few other examples of this in other files as well)

Reply:
Whoops, bad copy-pasta 😅 . Removed the duplicates

Reply:
😄  thank you!